examples/pylint/W0108_unnecessary_lambda.py

```diff
@@ -0,0 +1,10 @@
+from typing import List
+
+def add(lst: List[int]) -> int:
```

Comment:
This is the right idea but a bit complicated. Make this a simpler function, like `add_one`.